pkg/vtpm/Dockerfile

```diff
@@ -29,21 +26,31 @@ RUN ./configure --datarootdir=/usr/share/ && \
     make -j "$(getconf _NPROCESSORS_ONLN)" && make install
 
 WORKDIR /tpm2-tss
-ADD --keep-git-dir=true https://github.com/tpm2-software/tpm2-tss.git#4.0.1 /tpm2-tss
+ADD https://github.com/tpm2-software/tpm2-tss/tarball/2.3.1 /tpm2-tss.tgz
```

Comment:
What's the impact relative to e.g., https://nvd.nist.gov/vuln/detail/CVE-2023-22745 ?